README.adoc

```diff
@@ -238,7 +238,8 @@ fatal (i.e. the current step terminates with a failure):
 [NOTE]
```

Comment:
I think we need to reduce also text in the beginning and the long name:
Benedikt: a GitHub Action to check your code with [diKTat](https://github.com/saveourtool/diktat)

Here, for example, no need for a `Benedikt` word

Reply:
Addressed with c503257da552a3362366630b5c19d9010a09458d